README.md

```diff
@@ -14,18 +14,26 @@ A Simple, multi-profile Nix-flake deploy tool.
 
 Basic usage: `deploy [options] <flake>`.
 
-The given flake can be just a source `my-flake`, or optionally specify the node to deploy `my-flake#my-node`, or specify a profile too `my-flake#my-node.my-profile`. If your profile or node name has a `.` in it, simply wrap it in quotes, and the flake path in quotes (to avoid shell escaping), for example `'my-flake."myserver.com".system'`.
+Using this method all profiles specified in the given `<flake>` will be deployed (taking into account the [`profilesOrder`](#node)).
+
+Optionally the flake can be constraint to deploy just a single node (`my-flake#my-node`) or a profile (`my-flake#my-node.my-profile`).
+
+If your profile or node name has a . in it, simply wrap it in quotes, and the flake path in quotes (to avoid shell escaping), for example 'my-flake."myserver.com".system'.
+
+Any "extra" arguments will be passed into the Nix calls, so for instance to deploy an impure profile, you may use `deploy . -- --impure` (note the explicit flake path is necessary for doing this).
 
 You can try out this tool easily with `nix run`:
 - `nix run github:serokell/deploy-rs your-flake`
 
-Any "extra" arguments will be passed into the Nix calls, so for instance to deploy an impure profile, you may use `deploy . -- --impure` (note the explicit flake path is necessary for doing this).
+In you want to deploy multiple flakes or a subset of profiles with one invocation, instead of calling `deploy <flake>` you can issue `deploy --flake <flake> [<flake> ...]` where `<flake>` is supposed to take the same format as discussed before.
+
+Running in this mode, if any of the deploys fails, the deploy fill be aborted and all successful deploys revoked. `--revoke-succeeded false` can be used to override this behavior, otherwise the `auto-rollback` argument takes precedent.
```

Comment:
It might be good to explain what "revoke" means in this context here, also typo
```suggestion
Running in this mode, if any of the deploys fails, the deploy will be aborted and all successful deploys revoked. `--revoke-succeeded false` can be used to override this behavior, otherwise the `auto-rollback` argument takes precedent.
```

Reply:
> It might be good to explain what "revoke" means in this context here, also typo

Thinking about it, we should probably call it `rollback` to be in line with the existing terminology